indico/modules/events/editing/client/js/components/FileManager/reducer.js

```diff
@@ -0,0 +1,118 @@
+// This file is part of Indico.
+// Copyright (C) 2002 - 2019 CERN
+//
+// Indico is free software; you can redistribute it and/or
+// modify it under the terms of the MIT License; see the
+// LICENSE file for more details.
+
+import _ from 'lodash';
+import {combineReducers} from 'redux';
```

Comment:
Interesting, I didn't know you could use this for useReducer-based reducers as well! But I guess it makes sense since this is just a functional util with no ties to the redux store etc.

Reply:
Yeah, it's just a normal function.